go-controller/pkg/ovn/controller/egressservice/egressservice_zone.go

```diff
@@ -789,10 +807,10 @@ func (c *Controller) syncEgressService(key string) error {
 	}
 	allOps = append(allOps, createOps...)
 
-	if svcNodeInLocalZone && (len(v4RemoteToAdd)+len(v6RemoteToAdd)) > 0 {
+	if config.OVNKubernetesFeature.EnableInterconnect && svcNodeInLocalZone && (len(v4RemoteToAdd)+len(v6RemoteToAdd)) > 0 {
 		// when IC is disabled v[4|6]RemoteToRemove are empty and no ops are created
 		// with IC enabled, when service is hosted in the local zone, create static routes for remote endpoints
-		createOps, err = c.createOrUpdateLogicalRouterStaticRoutesOps(key, node.v4MgmtIP.String(), node.v6MgmtIP.String(), v4RemoteToAdd, v6RemoteToAdd)
+		createOps, err = c.createOrUpdateLogicalRouterPoliciesOps(key+"-ic", node.v4MgmtIP.String(), node.v6MgmtIP.String(), v4RemoteToAdd, v6RemoteToAdd)
```

Comment:
nit for the comment above which refers to routes instead of policies

Reply:
good catch, updated this.